### PR TITLE
fix: added star icon to message header

### LINKED ISSUE #267: Implement Star Icon in message header for Starred Messages
Messages that are starred currently do not have a distinct visual representation. We need to implement a star icon next to starred messages for clear differentiation. The message data received from the server already has the flag if the message is starred or not.

**Expected Behaviour**
star

### PR DESCRIPTION
**Pull request by anjy7**
# Brief Title
Added star icon to starred messages in the header

## Acceptance Criteria fulfillment

- [ ] Task 1
- [ ] Task 2
- [ ] Task 3

Fixes #267

## Video/Screenshots
https://www.loom.com/share/fe5b156a5a2a47f797395cac3a337351?sid=bb76e581-74c9-408b-8d78-ee3efd9cbb7d

### CODE REVIEW

**Comment by anjy7** [comment]
Hey @abhinavkrin , I have added star icon to the non sequential messages as shown is the ss. Just wanted to ask how to show the icon for the sequential messages?

**Comment by abhinavkrin** [comment]
Hey, @anjy7! Great work. 
You can add it in the place of header
Screenshot from 2023-10-19 23-36-36

File: packages/react/src/components/Message/MessageHeader.js [hunk 1]
```diff
@@ -57,6 +59,8 @@ const MessageHeaderTimestapCss = css`
   color: #9ea2a8;
 `;
 
+const authenticatedUserId = Cookies.get(RC_USER_ID_COOKIE);
```
Comment (abhinavkrin) [inline packages/react/src/components/Message/MessageHeader.js:62]: We need to get the authenticated user id from the userStore. We no longer use cookies for storing authenticated user ids.

**Review by abhinavkrin — CHANGES_REQUESTED** [review]
Use userStore to get authenticated user id.

File: packages/react/src/components/Message/MessageHeader.js (hunk 1 above)
Comment (anjy7) [reply packages/react/src/components/Message/MessageHeader.js:62]: Sure, I’ll make the changes

**Comment by anjy7** [comment]
Hey @abhinavkrin , apologies for the delay, got occupied with some other stuff. I have implemented the changes to make the component like RC when starred or sequential, please have a look :) 
image

**Comment by anjy7** [comment]
closed this PR and made a new one here - https://github.com/RocketChat/EmbeddedChat/pull/300

**Pull request closed without merging**